__tests__/tests.test.js

```diff
@@ -72,4 +74,81 @@ describe("/api/articles/:article_id", () => {
     })
   });
   
-
+//task 5
+describe("/api/articles/", () => {
+  test("get 200, then an array with all articles and sorted by created_at in descending order by default", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        expect(typeof (res.body)).toBe('object')
+        expect(res.body.articles.length).toBe(13)
+        expect(res.body.articles[0]).toMatchObject({
+          article_id: 3,
+          title: 'Eight pug gifs that remind me of mitch',
+          topic: 'mitch',
+          author: 'icellusedkars',
+          body: 'some gifs',
+          created_at: '2020-11-03T09:12:00.000Z',
+          votes: 0,
+          article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+          comment_count: '2'
+        })
```

Comment:
This is specifically testing that the first article looks like this, but we should be expecting every article to look like this instead, so a forEach here may be useful.